app/components/pilas-editor.js

```diff
@@ -7,14 +7,22 @@ export default Ember.Component.extend({
   cargando: true,
   showCode: false,
 
+  modoLecturaSimple: Ember.computed('model', function() {
+    return this.get('model.grupo.libro.modoLecturaSimple');
```

Comment:
Honestamente me hubiera gustado `this.get('model').modoLecturaSimple()` pero yafu.